nupic/encoders/multi.capnp

```diff
@@ -0,0 +1,51 @@
+@0xf07c1f24bf12fe13;
+
+using import "adaptivescalar.capnp".AdaptiveScalarEncoderProto;
+using import "category.capnp".CategoryEncoderProto;
+using import "coordinate.capnp".CoordinateEncoderProto;
+using import "date.capnp".DateEncoderProto;
+using import "delta.capnp".DeltaEncoderProto;
+using import "geospatial_coordinate.capnp".GeospatialCoordinateEncoderProto;
+using import "log.capnp".LogEncoderProto;
+using import "pass_through.capnp".PassThroughEncoderProto;
+using import "random_distributed_scalar.capnp".RandomDistributedScalarEncoderProto;
+using import "scalar.capnp".ScalarEncoderProto;
+using import "sdrcategory.capnp".SDRCategoryEncoderProto;
+using import "sparse_pass_through_encoder.capnp".SparsePassThroughEncoderProto;
+
+# Next ID: 14
+struct MultiEncoderProto {
+  struct IntType {
```

Comment:
Only pointer types are allowed in generics, and couldn't use `UInt32` directly as a parameter, so I needed to create the `IntType` wrapper.

Moot point, however, with bb546fd.